### DISCUSSION TITLE
Tokens that don't expire

### DISCUSSION THREAD

**Discussion by jeanlst** [fastapi-users/fastapi-users#859] [Q&A] [answered]
Hello, how would I structure my authentication to use permanent tokens?

**Comment by frankie567** [comment, marked as answer]
You can simply set the `lifetime_seconds` value to `None` in the [`JWTStrategy`](https://fastapi-users.github.io/fastapi-users/configuration/authentication/strategies/jwt/) or the [`DatabaseStrategy`](https://fastapi-users.github.io/fastapi-users/configuration/authentication/strategies/database/#strategy)
